src/game.cpp

```diff
@@ -316,6 +316,85 @@ static inline u8 getNeighbors(v3s16 p, INodeDefManager *nodedef, ClientMap *map,
 	return neighbors;
 }
 
+namespace {
```

Comment:
Otherwise it will have external linkage and have the potential to clash with classes with the same name.